docs/user/dashboard/share-dashboards.asciidoc

```diff
@@ -0,0 +1,27 @@
+[[share-dashboards]]
+== Share dashboards
+
+[[embedding-dashboards]]
+Share your dashboard outside of {kib}.
+
+From the *Share* menu, you can:
+
+* Embed the code in a web page. Users must have {kib} access
+to view an embedded dashboard.
+* Share a direct link to a {kib} dashboard
+* Generate a PDF report
+* Generate a PNG report
+
+TIP: To create a link to a dashboard by title, use: +
+`${domain}/${basepath?}/app/dashboards#/list?title=${yourdashboardtitle}`
```

Comment:
I'm not sure this tip is helpful without also explaining what each of these properties is, or giving examples